src/main/kotlin/com/pubnub/api/PubNub.kt

```diff
@@ -191,7 +192,9 @@ class PubNub(val configuration: PNConfiguration) {
         shouldStore: Boolean? = null,
         usePost: Boolean = false,
         replicate: Boolean = true,
-        ttl: Int? = null
+        ttl: Int? = null,
+        spaceId: SpaceId? = null,
```

Comment:
Will those parameters be not accessible by client?

I am thinking that sooner o latter we will create branch related to messageType and spaceId. Maybe we can create it now and store in it all changes related to this work?

Reply:
They will be available for users. Not sure what branch should I create. I guess what's missing is the kdoc for them